frontend/cypress/integration/featureFiles/wizard_k8sgw_api_routing.feature

```diff
@@ -52,14 +52,12 @@ Feature: Service Details Wizard: K8s Gateway API Routing
     Then user sees the "Istio Config" table with 2 rows
 
   @gateway-api
-  @wizard-k8s-routing
   @bookinfo-app
   Scenario: See a K8s Gateway generated with warning
     When user clicks in the "Istio Config" table "G" badge "reviews-gateway" name row link
     Then user sees the "kind: Gateway" regex in the editor
 
   @gateway-api
-  ## @wizard-k8s-routing
```

Comment:
@matejnesuta do you know why this was commented?

Reply:
Possibly because you don't have to open the wizard itself to delete the object. The scenario was written by @hhovsepy, maybe he knows better.